backend/app/services/knowledge/knowledge_service.py

```
@@ -312,15 +312,19 @@ def list_knowledge_bases(
+            # Group KBs (namespace != "default") are excluded — they are
+            # shown on the "Group" tab instead.
             # Personal: user_id matches and namespace is "default"
-            # Shared: id is in shared_kb_ids
-            # Bound: id is in bound_kb_ids (personal KBs bound to group chats)
+            # Shared: id is in shared_kb_ids (only personal KBs)
+            # Bound: id is in bound_kb_ids (only personal KBs bound to group chats)
             all_kbs = (
                 db.query(Kind)
                 .filter(
                     Kind.kind == "KnowledgeBase",
                     Kind.is_active == True,
```

Ruff (0.15.5)

[error] Avoid equality comparisons to `True`; use `Kind.is_active:` for truth checks

Replace with `Kind.is_active`

(E712)